### PR TITLE
Add links to discussions & discord

### PR DESCRIPTION
**Pull request by matheus23**
Discussion: https://github.com/wnfs-wg/spec/discussions/17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +23,23 @@ The specifications are:
 - [Namefilters](/spec/namefilter.md)
 - [Skip Ratchet](/spec/skip-ratchet.md)
 
+## Discussions & Community
+
+- For live chat, join the `#wnfs` channel in the [IPFS discord](https://discord.gg/vj7qWuAyHY)
+- For ideas & use cases, feel free to use this repo's [github discussions](https://github.com/wnfs-wg/spec/discussions/2)
```
Comment (expede) [inline README.md:29]: I wonder if this stuff would be better in the `wnfs-wg/.github` repo 🤔

File: README.md [hunk 2]
```diff
@@ -35,6 +23,23 @@ The specifications are:
...
+
+* [Brooklyn Zelenka](https://github.com/expede), [Fission](https://fission.codes)
+* [Philipp Krüger](https://github.com/matheus23), [Fission](https://fission.codes)
+
+## Authors
+
+* [Brooklyn Zelenka](https://github.com/expede), [Fission](https://fission.codes)
+* [Philipp Krüger](https://github.com/matheus23), [Fission](https://fission.codes)
+* [Daniel Holmgren](https://github.com/dholms), [Bluesky](https://blueskyweb.xyz/)
+* [Brendan O'Brien](https://github.com/b5), Number 0
```
Comment (expede) [inline README.md:41]: 👍

File: README.md (hunk 1 above)
Comment (expede) [reply README.md:29]: Screen Shot 2022-08-10 at 13 16 41

**Review by expede — APPROVED** [review]
LGTM! 👍 Just the one comment about removing either the discussion  section or the same from the `.github` repo

File: README.md (hunk 2 above)
Comment (expede) [inline README.md:41]:
```suggestion
* [Brendan O'Brien](https://github.com/b5), [number 0](https://www.n0.computer)
```

File: README.md [hunk 3]
```diff
@@ -35,6 +23,23 @@ The specifications are:
 - [Namefilters](/spec/namefilter.md)
 - [Skip Ratchet](/spec/skip-ratchet.md)
 
+## Discussions & Community
+
+- For live chat, join the `#wnfs` channel in the [IPFS discord](https://discord.gg/vj7qWuAyHY)
+- For ideas & use cases, feel free to use this repo's [github discussions](https://github.com/wnfs-wg/spec/discussions/2)
+
```
Comment (expede) [inline README.md:30]:
```suggestion

```

File: README.md (hunk 1 above)
Comment (matheus23) [reply README.md:29]: Ha! Awesome, I'll remove these lines from the spec.

**Pull request merged**